frontend/packages/dev-console/src/components/svg/SvgDropShadowFilter.tsx

```diff
@@ -19,24 +19,48 @@ const SvgDropShadowFilter: React.FC<SvgDropShadowFilterProps> = ({
   stdDeviation = 2,
   floodColor = '#030303',
   floodOpacity = 0.2,
-}) => (
-  <SVGDefs id={id}>
-    <filter
-      id={id}
-      x={`-${stdDeviation * 12.5}%`}
-      y={`-${stdDeviation * 12.5}%`}
-      width={`${100 + stdDeviation * 25}%`}
-      height={`${100 + stdDeviation * 25}%`}
-    >
-      <feDropShadow
-        dx={dx}
-        dy={dy}
-        stdDeviation={stdDeviation}
-        floodColor={floodColor}
-        floodOpacity={floodOpacity}
-      />
-    </filter>
-  </SVGDefs>
-);
+}) => {
+  if (window.navigator.userAgent.includes('Edge')) {
+    // feDropShadow is not supported by Edge
+    return (
+      <SVGDefs id={id}>
+        <filter
+          id={id}
+          x={`-${stdDeviation * 12.5}%`}
+          y={`-${stdDeviation * 12.5}%`}
+          width={`${100 + stdDeviation * 25}%`}
+          height={`${100 + stdDeviation * 25}%`}
+        >
+          <feGaussianBlur in="SourceAlpha" stdDeviation={stdDeviation} />
+          <feOffset dx={dx} dy={dy} />
+          <feMerge>
+            <feMergeNode />
+            <feMergeNode in="SourceGraphic" />
+          </feMerge>
```

Comment:
Your drop shadow is causing the node to look darker on edge.
I found this online that says it's equivalent of `feDropShadow`. https://docs.w3cub.com/svg/element/fedropshadow/
```
<feGaussianBlur in="alpha-channel-of-feDropShadow-in"
    stdDeviation="stdDeviation-of-feDropShadow"/>
<feOffset dx="dx-of-feDropShadow" dy="dy-of-feDropShadow"
    result="offsetblur"/>
<feFlood flood-color="flood-color-of-feDropShadow"
    flood-opacity="flood-opacity-of-feDropShadow"/>
<feComposite in2="offsetblur" operator="in"/>
<feMerge>
  <feMergeNode/>
  <feMergeNode in="in-of-feDropShadow"/>
</feMerge>
```

Reply:
Very well, I didn't spend a lot of time on it... Edge doesn't run smoothly, having darker shadows wasn't high on my fix list. I'll update it to be this and report back with a new screenshot.

Reply:
Fixed, updated the description image to showcase the result.